### DISCUSSION TITLE
Button

### DISCUSSION THREAD

**Discussion by GomesDev0001** [Auties00/WhatsappWeb4j#347] [General]
is it normal for the button to work only for whatsapp web?

like I use the web mode, when I send the button to the client (cell phone) nothing is sent, but when I enter the web I can see the button sent...

**Comment by Auties00** [comment]
> is it normal for the button to work only for whatsapp web?
> 
> like I use the web mode, when I send the button to the client (cell phone) nothing is sent, but when I enter the web I can see the button sent...

yeah unfortunately buttons don't work on - mobile anymore as Whatsapp has some restrictions in place.

**Comment by nexa-backend** [comment]
are the button messages already deprecated so the list message also ?
